example_legacy_tests/file-list-test.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test.js"></script>
+</head>
+<body>
+<input type="file" multiple onchange="onInputFileChange(this.files)">
+<div id="description"></div>
+<div id="console"></div>
+<script>
+description("Test the attribute of FileList.");
+
+debug("Start");
+
+function onInputFileChange(files)
+{
+    window.files = files;
+    shouldBeTrue("files instanceof FileList");
+    shouldBe("files.length", "2");
+    shouldBeTrue("files.item(0) instanceof File");
+    shouldBeEqualToString("files.item(0).name", "UTF8.txt");
+    shouldBeTrue("files.item(1) instanceof File");
+    shouldBeEqualToString("files.item(1).name", "UTF8-2.txt");
+    shouldBeNull("files.item(2)");
+    shouldBeNull("files.item(-1)");
+    shouldBeTrue("files.item(0) === files.item(4294967296)");
```

Comment:
This whole file was copy-pasted from a real test in the Chromium source. My intention was not to modify it at all to test my code with some examples of real tests. What do you think, should I still refactor?

Reply:
Ahh, didn't realize that. Maybe throw a comment at the top of the file pointing to where it came from then, and otherwise leave as-is? @Hexcles unless there are issues with c/p that code?

Reply:
Ahh sorry I missed this. We can add a comment or a README, but the directory name `testdata` should make it obvious that stuff in there are dummy test data.